src/Core/Style.js

```diff
@@ -160,6 +161,64 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
     style[category][name] = value;
 }
 
+/**
+ * @class
+ * @classdesc StyleContext stores metadata of one FeatureGeometry that are needed for its style computation:
+ * type of feature and what is needed (fill, stroke or draw a point, etc.) as well as where to get its
+ * properties and its coordinates (for base_altitude).
+ *
+ * @property {Coordinates}      worldCoord @private Coordinates of the FeatureGeometry in world system.
+ * @property {Coordinates}      localCoordinates @private Are the coordinates systeme origin local or global.
+ * @property {boolean}          isProjected @private Are the coordinates already been projected.
+ * @property {FeatureGeometry}  geometry  @private
+ * @property {Object}           globals Style type (fill, stroke, point, text and or icon) to consider. As
```

Comment:
`as well as` -> `also contains`

Reply:
corrected. And i'm thinking that we should really add a setzoom properties instead of putting in in global.

Reply:
I agree, maybe in another PR so we can move on with the one already open ?